.gitignore

```diff
@@ -1,2 +1,5 @@
 /.idea
+/.vs
+/.vscode
+
```

Comment:
Please don't, even `/.idea` should be removed from there

Reply:
done